main.py

```diff
@@ -49,8 +57,9 @@ def main():
     data = get_todays_games_json(todays_games_url)
     games = create_todays_games(data)
     data = get_json_data(data_url)
+    odds = get_odds()
     df = to_data_frame(data)
-    data, todays_games_uo, frame_ml, home_team_odds, away_team_odds = createTodaysGames(games, df)
+    data, todays_games_uo, frame_ml, home_team_odds, away_team_odds = createTodaysGames(games, odds, df)
```

Comment:
Try to run this with `python main.py -A`, I think the xgboost model path breaks with changes. Looks like a minor fix.